### DISCUSSION TITLE
DFU on Jumper T-Lite V2

### DISCUSSION THREAD

**Discussion by Shlipez** [RotorHazard/RotorHazard#975] [Q&A] [answered]
Please help me, I don't understand how to turn on the DFU on Jumper T-Lite V2, it says on the Internet to turn off the controller, hold down some horizontal buttons, press the on button and turn off the controller, I tried to do this, nothing worked, although a sign appeared on the screen with the inscription: ,,Key stuck."

**Comment by martinglass** [comment]
Is this really a question for the RotorHazard repo?

**Comment by HazardCreative** [comment, marked as answer]
This is the code repo of the RotorHazard race timer. You'll need to look for a community that better matches your question.
